app/src/main/java/com/rentminder/dto/Members.kt

```diff
@@ -1,6 +1,6 @@
 package com.rentminder.dto
 
-class Members (var membersId : Int = 0, var memberName : String) {
+data class Members (val membersId : Int = 0, val memberName : String) {
```

Comment:
Simple improvements but necessary ones. Make sure to use vals instead of vars in data classes as it is good practice to use immutable properties of instances. URL: https://www.digitalocean.com/community/tutorials/kotlin-data-class

Reply:
Agreed.